doc/reference/uefi_variables.md

```diff
@@ -0,0 +1,66 @@
+---
+discourse: ubuntu:42313
+---
+
+# UEFI variables for VMs
+
+{abbr}`UEFI (Unified Extensible Firmware Interface)` variables can be used by the boot loader and the operating system to configure early system startup.
```

Comment:
UEFI variables affect not only early startup. I would say that it's just a way of store and represent a UEFI configuration settings. So not only the UEFI firmware itself can modify/read these variables, but also a end-user using the operating system tools like `efibootmgr`, `efivar` in Linux, for example.